client/app/src/main/java/group01/smartcar/client/SimpleSpeedometer.java

```diff
@@ -0,0 +1,4 @@
+package group01.smartcar.client;
+
+public class SimpleSpeedometer{
+}
```

Comment:
This file is not used, I suggest we remove it.

Reply:
The unnecessary file has been deleted. I switched the MQTT connection to localhost. As you said it'll give better results.